spec/requests/admin/users_spec.rb

```diff
@@ -70,4 +70,62 @@
       end
     end
   end
+
+  describe "custom fees" do
```

Comment:
The e2e specs only seem to cover the admin part of these changes, can we add some specs for the actual checkout/purchase flow to verify correct fee is charged?